### DISCUSSION TITLE
Every time a new code is submitted, the github pages will be reset to the open state.

### DISCUSSION THREAD

**Discussion by kjxbyz** [community/community#65336] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

<img width="811" alt="2023-08-29 20 30 35" src="https://github.com/community/community/assets/47768002/0dda71d5-64d7-4910-92c7-3b1e5e04bde9">

**Comment by arafajs** [comment]
The bug that you are describing is caused by a setting called "Auto-deploy". When this setting is enabled, GitHub Pages will automatically deploy your changes to the production environment whenever you push to the gh-pages branch. However, if you have a custom domain configured for your GitHub Pages site, this setting can sometimes cause the custom domain to be reset to the "open" state.

To fix this bug, you can disable the "Auto-deploy" setting. To do this, follow these steps:

- Go to your repository's settings page.
- Click on the "Pages" tab.
- Under the "Deployment" section, uncheck the box next to "Auto-deploy".
- Click on the "Save" button.
